src/PublishCodeCovCoverageTask/utils/webUtils.ts

```
@@ -17,241 +17,260 @@
+          // Handle non-success status codes
+          if (response.status < 200 || response.status >= 300) {
+            return cleanup(new Error(`Failed to get '${fileUrl}' (${response.status})`), response);
+          }
 
-                    // Get total size from headers
-                    const contentLengthHeader = response.headers['content-length'];
-                    const totalBytes = contentLengthHeader !== undefined ? Number.parseInt(contentLengthHeader, 10) : NaN;
+          // Get total size from headers
+          const contentLengthHeader = response.headers['content-length'];
+          const totalBytes =
+            contentLengthHeader !== undefined ? Number.parseInt(contentLengthHeader, 10) : NaN;
```

Biome (1.9.4)

[error] Use Number.NaN instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.NaN instead.


(lint/style/useNumberNamespace)